data/strings/strings.txt

```diff
@@ -14519,6 +14519,41 @@
     zh-Hans = 楼层
     zh-Hant = 樓層
 
+  [level_value_0]
+    tags = android,ios
+    comment = Building level: ground floor
+    en = 0
+    de = Erdgeschoss
+    fr = Rez-de-chaussée
+
+  [level_value_1]
+    tags = android,ios
+    comment = Building level: one level above the ground
+    en = 1
+    de = 1. Stock
+    fr = 1er étage
+
+  [level_value_minus_1]
+    tags = android,ios
+    comment = Building level: one level below the ground
+    en = -1
+    de = 1. Untergeschoss
+    fr = 1er sous-sol
+
+  [level_value_negative]
+    tags = android,ios
+    comment = Building level: below the ground (at least 2 levels below)
+    en = -%d
+    de = %d. Untergeschoss
+    fr = %dème sous-sol
+
+  [level_value_positive]
+    tags = android,ios
+    comment = Building level: above the ground (at least 2 levels above)
+    en = %d
+    de = %d. Stock
+    fr = %dème étage
```

Comment:
Do I understand correctly that the "1er étage" / "2dème étage" difference could be solved by the plurals syntax?
E.g.
```
  [bookmarks_places]
    en:one = %d bookmark
    en:other = %d bookmarks
    fr:one = %d signet
    fr:other = %d signets
```
(and there are other options like `:few`, `:two`, probably French doesn't need them)

Reply:
e.g. `fr:one = Premier étage` might be possible also (need to check if validation allows it)